[Tue May 30 08:41:28.478073 2017] [:error] [pid 3433] [client 127.0.0.1:51510] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:41:28.478274 2017] [:error] [pid 3433] [client 127.0.0.1:51510] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:42:31.569924 2017] [:error] [pid 3437] [client 127.0.0.1:52032] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:42:31.570208 2017] [:error] [pid 3437] [client 127.0.0.1:52032] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:50:39.785530 2017] [:error] [pid 2785] [client 127.0.0.1:57032] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:50:39.786643 2017] [:error] [pid 2785] [client 127.0.0.1:57032] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:52:38.364886 2017] [:error] [pid 2787] [client 127.0.0.1:58114] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:52:38.365224 2017] [:error] [pid 2787] [client 127.0.0.1:58114] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:53:00.435601 2017] [:error] [pid 2788] [client 127.0.0.1:58310] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:53:00.435861 2017] [:error] [pid 2788] [client 127.0.0.1:58310] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:53:46.098604 2017] [:error] [pid 2789] [client 127.0.0.1:58690] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:53:46.099358 2017] [:error] [pid 2789] [client 127.0.0.1:58690] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:55:02.434588 2017] [:error] [pid 2785] [client 127.0.0.1:59484] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:55:02.435371 2017] [:error] [pid 2785] [client 127.0.0.1:59484] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:55:12.701821 2017] [:error] [pid 2786] [client 127.0.0.1:59602] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:55:12.702279 2017] [:error] [pid 2786] [client 127.0.0.1:59602] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:55:16.711586 2017] [:error] [pid 2787] [client 127.0.0.1:59636] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 08:55:16.712460 2017] [:error] [pid 2787] [client 127.0.0.1:59636] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 09:01:19.163870 2017] [:error] [pid 2787] [client 127.0.0.1:34614] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:01:19.192204 2017] [:error] [pid 2787] [client 127.0.0.1:34614] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:01:51.196435 2017] [:error] [pid 2788] [client 127.0.0.1:34916] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:01:51.197560 2017] [:error] [pid 2788] [client 127.0.0.1:34916] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:05:38.835311 2017] [:error] [pid 3433] [client 127.0.0.1:37098] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:05:38.835570 2017] [:error] [pid 3433] [client 127.0.0.1:37098] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:05:41.870058 2017] [:error] [pid 2785] [client 127.0.0.1:37130] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:05:41.870300 2017] [:error] [pid 2785] [client 127.0.0.1:37130] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:06:00.761382 2017] [:error] [pid 2786] [client 127.0.0.1:37348] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:06:00.761673 2017] [:error] [pid 2786] [client 127.0.0.1:37348] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:06:08.832496 2017] [:error] [pid 3437] [client 127.0.0.1:37414] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:06:08.832738 2017] [:error] [pid 3437] [client 127.0.0.1:37414] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:06:28.362064 2017] [:error] [pid 2786] [client 127.0.0.1:37610] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:06:28.362371 2017] [:error] [pid 2786] [client 127.0.0.1:37610] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:08:27.749311 2017] [:error] [pid 3433] [client 127.0.0.1:38886] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:08:27.749581 2017] [:error] [pid 3433] [client 127.0.0.1:38886] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:54:31.253951 2017] [:error] [pid 2789] [client 127.0.0.1:33768] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:54:31.254615 2017] [:error] [pid 2789] [client 127.0.0.1:33768] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:56:18.010831 2017] [:error] [pid 2788] [client 127.0.0.1:34680] PHP Fatal error:  Uncaught Error: Class 'App\\Exceptions\\Config' not found in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:72\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 72, referer: http://localhost:3000/auth/login
[Tue May 30 09:56:18.011179 2017] [:error] [pid 2788] [client 127.0.0.1:34680] PHP Fatal error:  Uncaught Error: Class 'App\\Exceptions\\Config' not found in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:72\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleShutdown()\n#4 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 72, referer: http://localhost:3000/auth/login
[Tue May 30 09:56:33.886827 2017] [:error] [pid 3437] [client 127.0.0.1:34816] PHP Fatal error:  Uncaught Error: Class 'App\\Exceptions\\Config' not found in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:72\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 72, referer: http://localhost:3000/auth/login
[Tue May 30 09:56:33.887751 2017] [:error] [pid 3437] [client 127.0.0.1:34816] PHP Fatal error:  Uncaught Error: Class 'App\\Exceptions\\Config' not found in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:72\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleShutdown()\n#4 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 72, referer: http://localhost:3000/auth/login
[Tue May 30 09:57:13.644015 2017] [:error] [pid 2789] [client 127.0.0.1:35142] PHP Fatal error:  Uncaught Error: Class 'App\\Exceptions\\Config' not found in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:72\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 72, referer: http://localhost:3000/auth/login
[Tue May 30 09:57:13.644461 2017] [:error] [pid 2789] [client 127.0.0.1:35142] PHP Fatal error:  Uncaught Error: Class 'App\\Exceptions\\Config' not found in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:72\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleShutdown()\n#4 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 72, referer: http://localhost:3000/auth/login
[Tue May 30 09:57:22.600052 2017] [:error] [pid 2787] [client 127.0.0.1:35218] PHP Fatal error:  Uncaught Error: Class 'App\\Exceptions\\Config' not found in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:72\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 72, referer: http://localhost:3000/auth/login
[Tue May 30 09:57:22.601801 2017] [:error] [pid 2787] [client 127.0.0.1:35218] PHP Fatal error:  Uncaught Error: Class 'App\\Exceptions\\Config' not found in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:72\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleShutdown()\n#4 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 72, referer: http://localhost:3000/auth/login
[Tue May 30 09:57:49.601368 2017] [:error] [pid 3433] [client 127.0.0.1:35450] PHP Fatal error:  Uncaught Error: Class 'App\\Exceptions\\Config' not found in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:72\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 72, referer: http://localhost:3000/auth/login
[Tue May 30 09:57:49.601862 2017] [:error] [pid 3433] [client 127.0.0.1:35450] PHP Fatal error:  Uncaught Error: Class 'App\\Exceptions\\Config' not found in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:72\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleShutdown()\n#4 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 72, referer: http://localhost:3000/auth/login
[Tue May 30 09:59:04.588287 2017] [:error] [pid 2789] [client 127.0.0.1:36086] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 09:59:04.588850 2017] [:error] [pid 2789] [client 127.0.0.1:36086] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 10:00:56.718519 2017] [:error] [pid 2788] [client 127.0.0.1:37036] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 10:00:56.719223 2017] [:error] [pid 2788] [client 127.0.0.1:37036] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 10:01:38.152555 2017] [:error] [pid 3437] [client 127.0.0.1:37416] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 10:01:38.155481 2017] [:error] [pid 3437] [client 127.0.0.1:37416] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 10:03:58.354920 2017] [:error] [pid 2788] [client 127.0.0.1:38590] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 10:03:58.355307 2017] [:error] [pid 2788] [client 127.0.0.1:38590] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://localhost:3000/auth/login
[Tue May 30 10:04:41.619068 2017] [:error] [pid 2789] [client 127.0.0.1:38878] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalThrowableError::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#3 {main}\n  thrown in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 10:04:41.619350 2017] [:error] [pid 2789] [client 127.0.0.1:38878] PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\\Component\\Debug\\Exception\\FatalErrorException::getStatusCode() in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(108): App\\Exceptions\\Handler->render(Object(Illuminate\\Http\\Request), Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#1 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(85): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->renderHttpResponse(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#2 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(119): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException))\n#3 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->hand in /var/www/html/laravel/admin-spa/app/Exceptions/Handler.php on line 50, referer: http://teste.com/auth/login
[Tue May 30 12:02:20.085102 2017] [:error] [pid 11070] [client 127.0.0.1:43964] PHP Fatal error:  Uncaught UnexpectedValueException: There is no existing directory at "/var/www/html/laravel/admin-spa/storage/logs" and its not buildable: Permission denied in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:171\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php(97): Monolog\\Handler\\StreamHandler->createDir()\n#1 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/AbstractProcessingHandler.php(37): Monolog\\Handler\\StreamHandler->write(Array)\n#2 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(337): Monolog\\Handler\\AbstractProcessingHandler->handle(Array)\n#3 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(616): Monolog\\Logger->addRecord(400, Object(UnexpectedValueException), Array)\n#4 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Log/Writer.php(203): Monolog\\Logger->error(Object(UnexpectedValueException) in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php on line 171, referer: http://localhost:3000/auth/login
[Tue May 30 12:02:20.125348 2017] [:error] [pid 11070] [client 127.0.0.1:43964] PHP Fatal error:  Uncaught UnexpectedValueException: There is no existing directory at "/var/www/html/laravel/admin-spa/storage/logs" and its not buildable: Permission denied in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:171\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php(97): Monolog\\Handler\\StreamHandler->createDir()\n#1 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/AbstractProcessingHandler.php(37): Monolog\\Handler\\StreamHandler->write(Array)\n#2 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(337): Monolog\\Handler\\AbstractProcessingHandler->handle(Array)\n#3 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(616): Monolog\\Logger->addRecord(400, Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException), Array)\n#4 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Log/Writer.php(203): Monolog\\Logger->error(Obj in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php on line 171, referer: http://localhost:3000/auth/login
[Tue May 30 12:02:22.566230 2017] [:error] [pid 3433] [client 127.0.0.1:44040] PHP Fatal error:  Uncaught UnexpectedValueException: There is no existing directory at "/var/www/html/laravel/admin-spa/storage/logs" and its not buildable: Permission denied in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:171\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php(97): Monolog\\Handler\\StreamHandler->createDir()\n#1 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/AbstractProcessingHandler.php(37): Monolog\\Handler\\StreamHandler->write(Array)\n#2 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(337): Monolog\\Handler\\AbstractProcessingHandler->handle(Array)\n#3 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(616): Monolog\\Logger->addRecord(400, Object(UnexpectedValueException), Array)\n#4 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Log/Writer.php(203): Monolog\\Logger->error(Object(UnexpectedValueException) in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php on line 171, referer: http://localhost:3000/auth/login
[Tue May 30 12:02:22.566678 2017] [:error] [pid 3433] [client 127.0.0.1:44040] PHP Fatal error:  Uncaught UnexpectedValueException: There is no existing directory at "/var/www/html/laravel/admin-spa/storage/logs" and its not buildable: Permission denied in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:171\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php(97): Monolog\\Handler\\StreamHandler->createDir()\n#1 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/AbstractProcessingHandler.php(37): Monolog\\Handler\\StreamHandler->write(Array)\n#2 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(337): Monolog\\Handler\\AbstractProcessingHandler->handle(Array)\n#3 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(616): Monolog\\Logger->addRecord(400, Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException), Array)\n#4 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Log/Writer.php(203): Monolog\\Logger->error(Obj in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php on line 171, referer: http://localhost:3000/auth/login
[Tue May 30 12:02:23.714276 2017] [:error] [pid 2787] [client 127.0.0.1:44062] PHP Fatal error:  Uncaught UnexpectedValueException: There is no existing directory at "/var/www/html/laravel/admin-spa/storage/logs" and its not buildable: Permission denied in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:171\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php(97): Monolog\\Handler\\StreamHandler->createDir()\n#1 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/AbstractProcessingHandler.php(37): Monolog\\Handler\\StreamHandler->write(Array)\n#2 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(337): Monolog\\Handler\\AbstractProcessingHandler->handle(Array)\n#3 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(616): Monolog\\Logger->addRecord(400, Object(UnexpectedValueException), Array)\n#4 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Log/Writer.php(203): Monolog\\Logger->error(Object(UnexpectedValueException) in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php on line 171, referer: http://localhost:3000/auth/login
[Tue May 30 12:02:23.720977 2017] [:error] [pid 2787] [client 127.0.0.1:44062] PHP Fatal error:  Uncaught UnexpectedValueException: There is no existing directory at "/var/www/html/laravel/admin-spa/storage/logs" and its not buildable: Permission denied in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:171\nStack trace:\n#0 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php(97): Monolog\\Handler\\StreamHandler->createDir()\n#1 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/AbstractProcessingHandler.php(37): Monolog\\Handler\\StreamHandler->write(Array)\n#2 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(337): Monolog\\Handler\\AbstractProcessingHandler->handle(Array)\n#3 /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Logger.php(616): Monolog\\Logger->addRecord(400, Object(Symfony\\Component\\Debug\\Exception\\FatalErrorException), Array)\n#4 /var/www/html/laravel/admin-spa/vendor/laravel/framework/src/Illuminate/Log/Writer.php(203): Monolog\\Logger->error(Obj in /var/www/html/laravel/admin-spa/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php on line 171, referer: http://localhost:3000/auth/login
